raiden/accounts.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import json
+import getpass
+
+from pyethapp.accounts import Account
+
+
+def find_datadir():
+    home = os.path.expanduser('~')
+    if home == '~':  # Could not expand user path
+        return None
+    datadir = None
+
+    if sys.platform == 'darwin':
+        datadir = os.path.join(home, 'Library', 'Ethereum')
+    # NOTE: Not really sure about cygwin here
+    elif sys.platform == 'win32' or sys.platform == 'cygwin':
+        datadir = os.path.join(home, 'AppData', 'Roaming', 'Ethereum')
+    elif os.name == 'posix':
+        datadir = os.path.join(home, '.ethereum')
+    else:
+        raise RuntimeError('Unsupported Operating System')
+
+    if os.path.isdir(datadir) is False:
```

Comment:
`is` is usually only used for `None`, more idiomatic would be `if os.path.isdir(datadir):`